src/codesign-validator.ts

```diff
@@ -46,19 +46,8 @@ export default class CodeSignValidator {
 
     validateMacSignature(fileExtension: string) {
         let promise = utility.defer();
-        let command = `spctl --assess -vv ${this.filePath} --type `;
+        let command = `codesign -dv --verbose=4 ${this.filePath}`;
```

Comment:
also, what sort of errors/exceptions do we expect from this tool?
are they handled correctly?

Reply:
the command is forked off as a child process. the stdout & stderr are printed out. Based on the return code we resolve/reject the promise